src/components/common.js

```diff
@@ -263,51 +255,10 @@ export const Select = ({ value, options, ...props }) => {
   }, props))
 }
 
-export const FadeBox = ({ fadePoint = '60%', style = {}, children }) => {
-  const {
-    paddingTop = '1.5rem',
-    paddingLR = '1.5rem',
-    borderRadius = '8px',
-    backgroundColor = 'rgba(255,255,255,0)',
-    borderColor = colors.gray[3],
-    ...containerStyle
-  } = style
-
+export const PageBox = ({ children, style = {} }) => {
```

Comment:
I kind of wanna keep these page fade boxes for now since without them it looks like there's slightly too much space between the top bar and the title of the page (i.e. Workspaces, Notebooks, etc.)

Reply:
Actually it looks fine--I think I was still just adjusting to the styling when I made this comment.